src/ray/core_worker/gcs_server_address_updater.cc

```diff
@@ -27,7 +27,7 @@ GcsServerAddressUpdater::GcsServerAddressUpdater(
       update_func_(update_func),
       updater_runner_(updater_io_service_),
       updater_thread_([this] {
-        SetThreadName("gcs_address_updater");
+        SetThreadName("gcs_address_upd");
```

Comment:
"gcs_addr_updater"

Reply:
It's still one character exceed, but not a big problem.